frontend/src/contexts/UploadContext.js

```diff
@@ -105,9 +98,28 @@ export default function UploadContextProvider(props) {
         uploadLoadedFiles,
         allFiles,
         setPersistenceScope,
+        invalidateCache,
       }}
     >
       {props.children}
     </UploadContext.Provider>
   )
 }
+
+/**
+ * Fetch the files in the gitea repo associated with project being updated
+ * @param repo{string}
+ * @param shouldFetch{boolean}
+ * @returns {{isLoading: boolean, isError: boolean, files: any | Array | null}}
+ */
+const useFetchRemoteRepoFiles = (repo, shouldFetch) => {
```

Comment:
This name is still too long.

Reply:
> Ignore the use of SWR in the uploadContext anyway its structure will change as I change the logic for the uploading process(will work on this tomorrow as it's more relevant to a different pr #62).

This file was deleted as part of #62, see https://github.com/kitspace/kitspace-v2/pull/62/commits/2cd8561d5113cc73ae528f2c4dd11dfd716e01d8.